src/mixins/events/index.js

```diff
@@ -10,8 +10,20 @@ export default {
      * @param {Event} event
      * @return {*|default.methods}
      */
-    propagate(event) {
+    propagateEvent(event) {
       return this.$emit(event.type, event);
     },
+
+    /**
+     * This method help propagate event values to
+     * parents components in vue.
+     * Its useful when you have input children components
+     * and want to use v-model in the parent component
+     * @param {Event} event
+     * @return {*|default.methods}
+     */
+    propagateValue(event) {
+      return this.$emit(event.type, event.target.value);
```

Comment:
```suggestion
      this.$emit(event.type, event.target.value);
```
I don't see reasons to return anything from here. Don't write unused code.

Reply:
updated